### PR TITLE
Conversions: Add light years, astronomical units, parsecs

### PR DESCRIPTION
**Pull request by jc86035**
## Description of new Instant Answer, or changes
This change adds (or should add, if I did it correctly) light years, astronomical units and parsecs to the Conversions IA. It also adds several aliases for the angstrom.

## Related Issues and Discussions
none

## People to notify
none

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/testing-reference/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/conversions/triggers.yml [hunk 1]
```diff
@@ -433,12 +433,61 @@ symbols: [yd]
...
+  - light-year
+  - light-years
+  - lichtjahr
+  - lichtjahre
+  - lichtjahren
+  - année-lumière
+  - années-lumière
+  - lichtjaar
+  - lichtjaren
+  type: length
```
Comment (kirkins) [inline share/goodie/conversions/triggers.yml]: Remove this indent and your PR will pass the tests.

Comment (jc86035) [inline share/goodie/conversions/triggers.yml]: Thanks, tried it.

**Comment by jc86035** [comment]
Incidentally, some units in triggers.yml don't have the `symbols: [KiB]` rows. Should all of the units have them (e.g. adding `symbols: [Å]` for ångström), or is this deliberate?

**Comment by mintsoft** [comment]
@pjhampton this LGTM, does it to you?

**Comment by pjhampton** [comment]
Reviewing this is on my todo list tomorrow @mintsoft
If you have tested the changes locally and not just skimmed the code then I say feel free to merge. 👍

**Comment by mintsoft** [comment]
I've only skimmed at the moment so I'll leave it up to you to sign off

**Comment by jc86035** [comment]
Who is supposed to add the tests?

**Comment by kirkins** [comment]
@jc86035 you'll have to edit this file:

https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/t/Conversions.t#L51

Basically you write tests based on pre-defined queries so duckpan can check that they return the expected results.

There are also tests which return undefined so you can check that something isn't querying:

https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/t/Conversions.t#L2717

So you should add tests related to the new features you added.

**Comment by jc86035** [comment]
@kirkins do I need to add more tests?

**Pull request merged**